accounts/models.py

```diff
@@ -85,3 +91,19 @@ def has_module_perms(self, app_label):
     @property
     def is_staff(self):
         return self.staff
+
+    def buy_stocks(self, quantity, price):
+        purchase_amount = Decimal(quantity) * price
+        if self.cash >= purchase_amount:
+            self.cash -= Decimal(quantity) * price
+            self.save()
+            return True
+        return False
+
+    def sell_stocks(self, quantity, price):
+        self.cash += Decimal(quantity) * price
+        self.save()
+
+    def update_cv(self, net_worth_list):
```

Comment:
This leaderboard ranking method may change in future so add a comment on top of this function saying TODO: this needs to be changed.